packages/feedback/src/app/modules/feedback/home/feedback-home.component.html

```diff
@@ -0,0 +1,139 @@
+          <div class="modal-container">
+            <div>
+              {{ feedbackDetails?.title
+                                ? (feedbackDetails?.title | ngXtruncate :80)
+                                : 'Feedback title not found'
+                              }}
+            </div>
+
+            <div class="modal-tag-group">
+              <span class="modal-tag modal-tag-module-name" *ngIf="feedbackDetails?.module">
+                {{ feedbackDetails?.module?.name }}
+              </span>
+              <span class="modal-tag modal-tag-open" *ngIf="feedbackDetails?.state === 'opened'">
+                open
+              </span>
+              <span class="modal-tag modal-tag-closed" *ngIf="!(feedbackDetails?.state === 'opened')">
+                closed
+              </span>
+            </div>
+
+            <div class="feedback-header-information">
+              <div class="feedback-created-by">
+                <div class="fi-modal-headings">
+                  Created By
+                </div>
+                <div>
+                  <span class="fi-modal-bold">
+                    {{ feedbackDetails?.timestamp?.createdBy?.name }}
+                  </span>
+                  at {{ feedbackDetails?.timestamp?.createdAt | date: 'dd-MMM-yyyy' }}
+                </div>
+              </div>
+
+              <div class="feedback-assignees">
+                <div class="fi-modal-headings">Assignee(s)</div>
+                <span *ngFor="let assignee of feedbackDetails?.assignees">
+                  <a target="blank" *ngIf="assignee?.name" href="{{ assignee?.web_url }}">
+                    {{ assignee?.name }}
+                  </a>
+                </span>
+                <span *ngIf="!(feedbackDetails?.assignees?.length)" class="no-feedback-description">Not Assigned</span>
+              </div>
+
+              <div class="feedback-experience">
+                <div class="fi-modal-headings">Experience</div>
+                <div class="fi-modal-bold">
+                  {{ (feedbackDetails?.experience)
+                          ? feedbackDetails?.experience
+                          : 'N/A'
+                          }}
+                </div>
+              </div>
+            </div>
+
+            <div class="feedback-modal-bold">
+              <div class="fi-modal-headings">
+                Description
+              </div>
+              <div class="fi-modal-bold">
+                <p *ngIf="feedbackDetails?.description; else noFeedbackDescription">{{ feedbackDetails.description }}</p>
+                <ng-template #noFeedbackDescription class="no-feedback-description">
+                  Description not found
+                </ng-template>
+              </div>
+
+              <div class="fi-modal-footer">
+                <a class="btn fb-issue-link" *ngIf="feedbackDetails?.iid"
+                  href="https://gitlab.cee.redhat.com/one-portal/one-portal-client/issues/{{ feedbackDetails?.iid }}"
```

Comment:
Please remove this internal gitlab host information

Reply:
okay, if i will remove it then how gitlab link will get open for a particular issue.